src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonArray.cs

```diff
@@ -113,9 +113,10 @@ internal int GetElementIndex(JsonNode? node)
         }
 
         /// <summary>
-        /// Returns enumerator that wraps calls to <see cref="JsonNode.GetValue{T}"/>.
+        /// Returns an enumerable that wraps calls to <see cref="JsonNode.GetValue{T}"/>.
```

Comment:
This change didn't get reflected in the dotnet-api-docs PR, FYI. APIs with existing docs don't get updated by the porting tool.

Reply:
Fixed.